extras/travis_build.sh

```diff
@@ -215,82 +219,83 @@ if [ "$xSTATIC_ANALYSIS" = "1" ]; then
     "$IGNORE_MSG_VERA" "$IGNORE_MSG_CPPCHECK" "$IGNORE_MSG_CLANG_FORMAT" "$IGNORE_MSG_PEP8"
 else
     echo "MSGBLD0225: Static code analysis skipped due to build configuration."
-fi  # End of Static code analysis.
```

Comment:
If this is moved to the bottom of the script, execution of the build will depend on the setting of xSTATIC_ANALYSIS. Wouldn't it be better to leave control to the build matrix in the .travis.yaml file and add another variable to the matrix and encapsulate the build process?

Reply:
There seems to be a variable `xRUN_TESTSUITE` that already follows exactly this pattern (it is 1 if and only if `xSTATIC_ANALYSIS` is 0). Right now it is only wrapping the `make installcheck` command. Does it still make sense to have an extra variable (e.g. `xBUILD_NEST`), or shall we rename `xRUN_TESTSUITE` and make it encapsulate *both* the build and running the test suite?

Reply:
`xRUN_TESTSUITE` skips `make installcheck` if set to 0. In contrast, `xSTATIC_ANALYSIS` skips code analysis and turns the build on if set to 0. This is not what I would expect when looking at the build matrix in the `.travis.yml` file.

What do you think of the following solution? 
- rename `xRUN_TESTSUITE` into `xRUN_BUILD_AND_TESTSUITE` and change the script ...

```
if [ "$xRUN_BUILD_AND_TESTSUITE" = "1" ]; then
  echo
  echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"  
  echo "+               B U I L D   N E S T                                           +"
  echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
   echo "MSGBLD0250: Running Make."
  make VERBOSE=1
  echo "MSGBLD0260: Make completed."
  ...
  make install
  ...
  make installcheck
fi
```
Test without build does not make any sense. We can combine it and be more explicit without hiding the logic.